<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Training Cifar10 Classifier Using VGG16</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>19</y>
      <width>341</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>VGG16 TEST</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1. Show Train Images</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>2. Show HyperParameter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>3. Show Model Shortcut</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>4. Show Accuracy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>440</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5. Test</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
